luminaire/model/window_density.py

```diff
@@ -512,6 +515,7 @@ def _get_result(self, input_df=None, detrend_order=None, agg_data_model=None, va
         :param float anomaly_scores_gamma_alpha: Gamma fit alpha parameter.
         :param float anomaly_scores_gamma_loc: Gamma fit location parameter.
         :param float anomaly_scores_gamma_beta: Gamma fit beta parameter.
+        :param agg_data: Aggregated Data per day.
```

Comment:
Same comment as before

Reply:
This is a higher level aggregated data which is per day at this moment